app/src/organisms/ErrorRecoveryFlows/hooks/useDeckMapUtils.ts

```diff
@@ -282,13 +282,18 @@ export const getRunCurrentModulesInfo = ({
     return runRecord.data.modules.reduce<RunCurrentModuleInfo[]>(
       (acc, module) => {
         const moduleDef = getModuleDef(module.model)
+        const moduleType = getModuleType(moduleDef.model)
 
-        // Get the labware that is placed on top of the module.
+        // for stacker, we only want to consider labware in the hopper as "nested"
```

Comment:
Very small nit: Can we keep the original comment explaining what `nestedLabware` means? It otherwise wouldn't be clear to me.

```suggestion
        // Get the labware that is placed on/in the module.
        // for stacker, we only want to consider labware in the hopper as "nested"
```

Reply:
Yep!